Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM postgres:9.6
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        build-essential \
+        git \
+        postgresql-server-dev-9.6 && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
This is going to still leave around a bunch of stuff we don't need in the final image — eventually we should do the `make install` as part of this `RUN` command and `apt-get purge` everything at the end.